pkg/validations/validations.go

```diff
@@ -52,6 +52,27 @@ func validateMachineTypeDataTypes(resource map[string]interface{}) error {
 	return nil
 }
 
+func validateRegionDataTypes(resource map[string]interface{}) error {
+	resourceId := resource[Id].(string)
+	if CloudProviderId == "" {
+		return errors.UserErrorf(errormsgs.EmptyRequiredField, resourceField, resourceId)
+	}
```

Comment:
This is causing a compilation error:
```
[rcampos@rcampos-thinkpadt14sgen2i ocm-csv-parser]$ go build ./
# github.com/openshift-online/ocm-csv-parser/pkg/validations
pkg/validations/validations.go:58:58: undefined: resourceField
```
 I'm not sure what is being checked here, as CloudProviderId is a constant

Reply:
Fixed and tests pass